INSTALL.rst

```diff
@@ -12,29 +12,16 @@ provided with the source.
 Dependencies
 ************
 
-A full list of the Cyclus package dependencies is shown below:
-
-====================   ==================
-Package                Minimum Version
-====================   ==================
-``CMake``                2.8
-``boost``                1.46.1
-``libxml2``              2
-``libxml++``             2.36
-``python``               2.7 or 3.3+
-``sqlite3``              3.7.10
-``HDF5``                 1.8.4
-``Coin-Cbc``             2.5
-====================   ==================
+A full list of the Cyclus package dependencies is listed :doc:`here <DEPENDENCIES>`:
```

Comment:
Instead of pointing to the file, we should be able to have  like what's in line 20 (`.. website_start`) as a pointer around the table in `DEPENDENCIES.rst` that we can pull it. It'll probably be cleaner when building the website while also preventing from having to update multiple tables.

Reply:
I think we're caught in a github rendering vs sphinx/website rendering bind.  I initially suggested the include like you suggested, but it doesn't render at all on github and looks like gibberish.  On the other hand, this sphinx based reference also doesn't render (although doesn't look like gibberish).  So maybe we go with the include?

Reply:
Do you have any screenshots of how the build looks and the code for it? If not, maybe we can hop on a call so I can see what you're getting.

Reply:
You can see how the `INSTALL` file renders on github when using the include approach [here](https://github.com/cyclus/cyclus/blob/5e2eed6781d2732dc0e8e70328055f4b05ffb3f1/INSTALL.rst) (this is from an old commit).

Notice how the section on "Dependencies" seems incomplete and then take a look at the raw code.

![Screenshot 2024-03-28 at 11 13 19](https://github.com/cyclus/cyclus/assets/764326/d26fa139-9e10-471d-8ade-5c3c92430350)

Reply:
I see what you mean, and this is what I had in mind for code to use. Just pointing to it via what we have now should be good then.